internal/commands/create.go

```diff
@@ -200,22 +196,21 @@ func getConstraint(violation rego.Rego) (unstructured.Unstructured, error) {
 
 	matchers, err := violation.Matchers()
 	if err != nil {
-		return constraint, err
+		return unstructured.Unstructured{}, fmt.Errorf("matchers: %w", err)
 	}
 	if len(matchers.KindMatchers) == 0 {
-		return constraint, nil
+		return unstructured.Unstructured{}, nil
```

Comment:
I don't think we want to do this. If KindMatchers aren't used, that just means the policy author wants the constraint to apply to all resources.

Reply:
.. yeah my bad, awesome catch. I'll do a follow-up after this to create some additional configuration tests.